transaction/core/src/amount/error.rs

```diff
@@ -1,9 +1,11 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
 //! Errors that can occur when handling an amount commitment.
 
 use displaydoc::Display;
 
 /// An error which can occur when handling an amount commitment.
-#[derive(Debug, Display, Eq, PartialEq)]
+#[derive(Debug, Display, Eq, PartialEq, Clone)]
```

Comment:
Nit: Prefer keeping Derive's sorted.
```suggestion
#[derive(Clone, Debug, Display, Eq, PartialEq)]
```